SamMou.Api.Graph/GraphQLSchema.cs

```diff
@@ -9,9 +9,10 @@ namespace SamMou.Api.GraphQL
 {
     public class GraphQLSchema : Schema
     {
-        public GraphQLSchema(IServiceProvider services, GraphQLQuery query) : base(services)
+        public GraphQLSchema(QueryObject query, MutationObject mutation, IServiceProvider sp) : base(sp)
         {
-            Query = query;
+                Query = query;
+                Mutation = mutation;
```

Comment:
I saw this in graphql-dotnet documentation, what do you think of this pattern? I think this is cleaner so we don't clutter QueryObject and MutationObject into app.startup? (if it is indeed from startup). In the future we might add `SubscriptionObject`, then we need to add in startup again?

```csharp
public class StarWarsSchema : Schema
{
    public StarWarsSchema(IServiceProvider serviceProvider)
        : base(serviceProvider)
    {
        Query = serviceProvider.GetRequiredService<StarWarsQuery>();
        Mutation = serviceProvider.GetRequiredService<StarWarsMutation>();
    }
}
```

Reply:
https://graphql-dotnet.github.io/docs/getting-started/dependency-injection

Reply:
![image](https://user-images.githubusercontent.com/26915914/121793478-eb6a2d00-cc31-11eb-9b9f-f764c2a6059d.png)

Reply:
it's the same approach in order for you to getrequiredservice, you will need to addscope/addsingleton in your startup.cs, the differences is just whether you wants to inject in the constructor or you getrequiredservice